code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -38,6 +37,66 @@
 				break
 	return 1
 
+//////////////////////////////////////////////
+//                                          //
+//               CHALLENGERS                ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/challengers
+	name = "Syndicate Challengers"
+	role_category = /datum/role/traitor/challenger
+	protected_from_jobs = list("Security Officer", "Merchant", "Warden", "Head of Personnel", "Cyborg", "Detective",
+							"Head of Security", "Captain", "Chief Engineer", "Chief Medical Officer", "Research Director")
+	restricted_from_jobs = list("AI","Mobile MMI")
+	required_candidates = 2
+	weight = 1
+	cost = 15
+	var/traitor_threshold = 3
```

Comment:
This is a straight copy and paste of the traitor ruleset. Can't you inherit from the original ruleset somehow ?

Reply:
Just because most of its values beside the cost are the same doesn't mean we won't want to independently adjust them down the road. I think rulesets should be fully independent from one another for that reason.